tests/integration/test_dynamodb.py

```diff
@@ -506,6 +506,31 @@ def wait_for_stream_created(table_name):
         )
         assert "ShardIterator" in response
 
+    def test_dynamodb_partiql_missing(self):
+        dynamodb = aws_stack.create_external_boto_client("dynamodb")
```

Comment:
nit: No change required for now, but in the future we could use the `dynamodb_client` pytest fixture. 👍  (which will also come in handy if we perform parity tests against real AWS...)